recipes/photutils/meta.yaml

```diff
@@ -20,12 +20,13 @@ requirements:
   build:
     - python
     - setuptools
-    - numpy x.x
+    - numpy 1.9.*  # [not py36]
```

Comment:
Is 1.9 available on python 3.6? Right, I see it there now. Pushing the change...